lib/src/postgrest_query_builder.dart

```diff
@@ -154,16 +195,29 @@ class PostgrestQueryBuilder<T> extends PostgrestBuilder<T, T> {
 
   /// Performs a DELETE on the table.
   ///
-  /// By default the deleted record(s) will be returned. Set [returning] to minimal if you don't need this value.
+  /// By default no data is returned. Use a trailing `select` to return data.
+  ///
+  /// Default (not returning data):
   /// ```dart
-  /// postgrest.from('messages').delete().eq('message', 'foo')
+  /// await supabase
+  ///     .from('messages')
+  ///     .delete()
+  ///     .eq('message', 'foo');
+  /// ```
+  ///
+  /// Returning data:
+  /// ```dart
+  /// await supabase
+  ///     .from('messages')
+  ///     .delete()
+  ///     .eq('message', 'foo')
+  ///     .select();
   /// ```
   PostgrestFilterBuilder<T> delete({
-    ReturningOption returning = ReturningOption.representation,
```

Comment:
I think it's fine that we don't fetch the deleted data, because the return type is `void`, but removing this parameter (although it's not used) may cause compile time error. Maybe mark it as `deprecated` instead?

Reply:
Sure, sounds good!